src/ui/buttons/buttonDesigns/switch-logo-and-show-label-text.jsx

```diff
@@ -0,0 +1,164 @@
+/* @flow */
+/** @jsx node */
+import { LOGO_CLASS, PPLogo } from '@paypal/sdk-logos/src';
+import { node, type ChildType } from 'jsx-pragmatic/src';
+
+import { CLASS } from '../../../constants';
+import { BUTTON_SIZE_STYLE } from '../config';
+
+import type { ContentOptions } from './types';
+
+export const SWITCH_LOGO_AND_SHOW_LABEL_CONFIG = {
```

Comment:
If you are going to define a constant for this there's no need to define it again here

Reply:
I already create a reusable constant that is gonna be reused every where. see the constants file.